tools/data-handler/test/command-handler.test.ts

```diff
@@ -1656,8 +1655,10 @@ describe('move command', () => {
     expect(result.statusCode).to.equal(200);
   });
   it('move card to another card (success)', async () => {
-    const sourceId = 'decision_11';
-    const destination = 'decision_10';
+    const cards = await new Show().showProjectCards(options.projectPath!);
```

Comment:
Ensure that there are at least two cards in the array.

Reply:
Fixed in [811d7c2](https://github.com/CyberismoCom/cyberismo/pull/127/commits/811d7c26427bc63e714c5db054551d000e028a08)